sematic/plugins/building/docker_client_utils.py

```diff
@@ -0,0 +1,72 @@
+"""
+Utility code for interacting with the `docker-py` client.
+"""
+# Standard Library
+import sys
+import textwrap
+from typing import Any, Dict, Generator, List, Optional, TextIO
+
+_ROLLING_PRINT_COLUMNS = 120
+_ROLLING_PRINT_ROWS = 10
+
+
+def _rolling_print_status_updates(
+    status_updates: Generator[Dict[str, Any], None, None],
+    output_handle: TextIO = sys.stderr,
```

Comment:
I think `sys.stderr` might technically be mutable, if the code does any manual messing around with file handle redirection. So this should probably be set to `None` here, and then `None` should be treated as `sys.stderr` in the function body.

Reply:
Done.